src/python/grpcio/grpc/_typing.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Common types for gRPC Sync API"""
+
+from typing import Any, Callable, Iterable, Sequence, Tuple, TypeVar, Union
+
+from grpc._cython.cygrpc import EOF
+
+RequestType = TypeVar('RequestType')
+ResponseType = TypeVar('ResponseType')
+SerializingFunction = Callable[[Any], bytes]
```

Comment:
SGTM

Reply:
Tried change the type of SerializingFunction from `Callable[[Any], bytes]` to `Callable[[RequestType], bytes]`, but looks like it's not necessary.

For example, in [_Channel.unary_unary](https://github.com/grpc/grpc/blob/89de6312cd949fdd2aeaeaccc313fb74ac0a3853/src/python/grpcio/grpc/_interceptor.py#L492-L501) the annotation only appears in the signature thus not enforce anything, and pytype also complains with error: `Invalid type annotation 'RequestType'  [invalid-annotation] Appears only once in the signature`.

I think the best way is just keep it as `Any` since it's also what we have in [aio stack](https://github.com/grpc/grpc/blob/89de6312cd949fdd2aeaeaccc313fb74ac0a3853/src/python/grpcio/grpc/aio/_typing.py#L27,L28).